content/hardware/06.nicla/boards/nicla-voice/tutorials/glass-break-detector/content.md

```diff
@@ -0,0 +1,838 @@
+
+### Model Testing
+
+After training, the model is evaluated using a test dataset that includes previously unseen audio samples. This evaluation helps ensure the model generalizes adequately to new data and determine glass-breaking events under various conditions.
+
+![Model Testing Results](assets/edge-model-test.png)
+
+The image above shows the testing results with an accuracy of **99.12 %**. This can be found within the **Model testing** section in the Edge Impulse platform.
+
+The confusion matrix highlights the model's capability to classify audio samples correctly while identifying potential areas for improvement:
+
+- Correctly classified "glassbreak" events: 94.4 %
+- Correctly classified "unknown" samples: 100 %
+
+Misclassifications could have happened for short-duration glass-breaking sounds, suggesting the need for additional data or refinement of the processing pipeline to improve robustness. Regular updates to the dataset with new samples can address such limitations and improve performance.
+
+***The model performance can be affected if the application is implemented in a very different environment than the one used for training. It is recommended that the datasets be provided with new samples and the model be retrained for a new and upgraded deployment.***
+
+## Deployment
+
+### Model Deployment
+
+After training the model, it can be deployed to the Nicla Voice using **binary firmware builder from Edge Impulse**. This builder creates firmware that includes the trained model and the necessary scripts for on-device inference, allowing operation without internet connectivity.
+
+To begin, navigate to the **Deployment** section in the Edge Impulse platform. The following image shows the deployment configuration screen, where you can select Nicla Voice as the deployment target.
+
+![Configure Deployment for Nicla Voice](assets/edge-deployment-target.png)
+
+Once the Nicla Voice is selected as the deployment target, click on **Build** to start the firmware build process. The animation below provides an overview of how this process looks:
+
+![Deployment Build for Nicla Voice](assets/nicla-edge-impulse-fw-build-process.gif)
+
+After the build is complete, the firmware files will automatically download. You can retrieve them later from the **Latest build** tab in the **Deployment** section if needed.
+
+![Nicla Voice Firmware Build Files](assets/edge-fw-build.png)
+
+The downloaded firmware comes as a compressed file. Extract it to your preferred directory. You will find several files in the extracted folder, including a directory named **`ndp120`**.
+
+This folder contains the NDP120 processor firmware and the trained machine learning model. There are four scripts for each supported operating system:
+
+- **flash_<os>:** Runs the MCU firmware and model flash scripts.
```

Comment:
Using <> in the markdown is problematic, this is why the local preview doesn't work also the github preview, I changed to this: and it fixes all the problems. 

- **flash_(os):** Runs the MCU firmware and model flash scripts.
- **flash_(os)_mcu:** Flashes only the MCU firmware.
- **flash_(os)_model:** Flashes the NDP firmware, DSP firmware (if not present), and updates the model.
- **format_(os)_ext_flash:** Erases the external flash memory.

**`(os)`** refers to the operating system: linux, mac, or windows.

Reply:
ah okay, thanks! Applying the changes